clang/lib/Headers/hlsl/hlsl_intrinsic_helpers.h

```diff
@@ -101,6 +101,16 @@ constexpr vector<T, N> smoothstep_vec_impl(vector<T, N> Min, vector<T, N> Max,
 #endif
 }
 
+template <typename T> constexpr vector<T, 4> lit_impl(T NDotL, T NDotH, T M) {
+  bool DiffuseCond = NDotL < 0;
+  T Diffuse = select<T>(DiffuseCond, 0, NDotL);
+  vector<T, 4> Result = {1, Diffuse, 0, 1};
+  bool SpecularCond = or (DiffuseCond, (NDotH < 0));
```

Comment:
```suggestion
  bool SpecularCond = or(DiffuseCond, (NDotH < 0));
```

clang-format adds an undesirable space here. This will eventually be fixed but you should remove it manually for now.

Reply:
Yeah I noticed that—won't the manual change make my clang-format check fail though?

Reply:
It shouldn't.

Reply:
I think deric means you need to tell clang format to ignore it for now.

Reply:
I had a similar issue when implementing the `and` intrinsic (#127098). Justin filed an issue for it (#127851) and in the meantime I manually fixed the spacing. I don't recall the PR code format check complaining about it.

Reply:
Hmm. Seems to be complaining this time for some reason.

Reply:
Adding the manual space is making it fail :\ Should I tell clang-format to ignore it? And how do I do that?

Reply:
It should be fine to leave it though.

Reply:
Oh wait. Add this above the line
```
// clang-format off
```

Reply:
Code updated